src/config.js

```diff
@@ -135,6 +135,24 @@ export function loadJSConfigFile(filePath: string): Object {
   return configObject;
 }
 
+export function loadJSONConfigFile(filePath: string): Object {
```

Comment:
Adding a new function for this is not necessary. You can re-use the same config loading function. You may need to make some adjustments to `loadJSConfigFile` to handle the JSON.